core/capabilities/integration_tests/por/fetchtrueusd_test.go

```diff
@@ -68,7 +68,7 @@ func Test_FetchTrueUSDWorkflow(t *testing.T) {
 	framework.CreateWasmBinary(t, mainFile, wasmFile)
 
 	triggerSink := framework.NewTriggerSink(t, "cron-trigger", "1.0.0")
-	dataFeedsCacheContract := setupDons(ctx, t, lggr, wasmFile, "*/5 * * * * *", triggerSink)
```

Comment:
so if I understand the existing POR test has been 'hijacked' to test workflow update, fine for the hot fix, but could this be followed up with a test to explicity test workflow update that is seperate from the POR test? (reusing where necessary)